rfcs/spinner-component.md

```diff
@@ -0,0 +1,30 @@
+# Spinner
```

Comment:
for accessibility it seems like adding `role="progressbar"` should be sufficient.